index.js

```diff
@@ -91,77 +93,10 @@ document.getElementById('card-btn').onclick = () => {
-  document.getElementById('site').href = projectprop[1].site;
-  document.getElementById('source').href = projectprop[1].source;
-};
-
-document.getElementById('card-btn3').onclick = () => {
-  document.getElementById('pop-up').classList.add('star');
-  document.getElementById('main').classList.add('blur');
-  document.getElementById('Portfolio').classList.add('blur');
-  document.getElementById('About').classList.add('blur');
-  document.getElementById('Contact').classList.add('blur');
-  document.getElementById('l1').innerHTML = projectprop[1].technologies[0];
-  document.getElementById('l2').innerHTML = projectprop[1].technologies[1];
-  document.getElementById('l3').innerHTML = projectprop[1].technologies[2];
-  document.getElementById('pop').src = projectprop[1].URL;
-  document.getElementById('h2').innerHTML = projectprop[1].name;
-  document.getElementById('site').href = projectprop[1].site;
-  document.getElementById('source').href = projectprop[1].source;
-};
-
-document.getElementById('card-btn4').onclick = () => {
-  document.getElementById('pop-up').classList.add('star');
-  document.getElementById('main').classList.add('blur');
-  document.getElementById('Portfolio').classList.add('blur');
-  document.getElementById('About').classList.add('blur');
-  document.getElementById('Contact').classList.add('blur');
-  document.getElementById('l1').innerHTML = projectprop[1].technologies[0];
-  document.getElementById('l2').innerHTML = projectprop[1].technologies[1];
-  document.getElementById('l3').innerHTML = projectprop[1].technologies[2];
-  document.getElementById('pop').src = projectprop[1].URL;
-  document.getElementById('h2').innerHTML = projectprop[1].name;
-  document.getElementById('site').href = projectprop[1].site;
-  document.getElementById('source').href = projectprop[1].source;
-};
-
-document.getElementById('card-btn5').onclick = () => {
-  document.getElementById('pop-up').classList.add('star');
-  document.getElementById('main').classList.add('blur');
-  document.getElementById('Portfolio').classList.add('blur');
-  document.getElementById('About').classList.add('blur');
-  document.getElementById('Contact').classList.add('blur');
-  document.getElementById('l1').innerHTML = projectprop[1].technologies[0];
-  document.getElementById('l2').innerHTML = projectprop[1].technologies[1];
-  document.getElementById('l3').innerHTML = projectprop[1].technologies[2];
-  document.getElementById('pop').src = projectprop[1].URL;
-  document.getElementById('h2').innerHTML = projectprop[1].name;
-  document.getElementById('site').href = projectprop[1].site;
-  document.getElementById('source').href = projectprop[1].source;
+document.getElementById('submit-form').onsubmit = (e) => {
+  const email = document.getElementById('email').value;
+  if (email !== email.toLowerCase) {
+    e.preventDefault();
+    document.getElementById('error-message').innerHTML = 'Email must be lowercase';
+  }
```

Comment:
- [ ] Kindly make sure the form is submitted if the email check is correct by calling the `submit fx` in an else statement. because as you may notice you are only preventing the form to be submitted no matter what.

![image](https://user-images.githubusercontent.com/97100091/210656878-046be6d8-c47c-4142-b1ad-eeba15a61856.png)